.github/workflows/helm-generate-rbac.yaml

```diff
@@ -0,0 +1,41 @@
+name: Generate RBAC from Helm charts
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  generate-rbac:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: copy working directory to GOPATH
+      run: sudo mkdir -p /home/runner/go/src/github.com && sudo cp -a /home/runner/work/rook /home/runner/go/src/github.com/
+
+    - name: run helm-gen-rbac
```

Comment:
Do we need a new action for this, or could it just be another step in the build action we already have? It's quick to run, right?

Reply:
Which action do you mean?

Reply:
Currently, all the different validations are separate actions, and I follow that pattern here. I generally think that we could have them be all in one action, but maybe that's a future change?

Reply:
I see this validation as similar to `make codegen` and `make crds`, which run as part of the build.yml workflow, so this could potentially be another validation step in there too.

Reply:
Both codegen and crds have their own individual actions as well as being included in build.yml (for testing on  mac). But good point to note that I forgot to include it on the mac side.

Reply:
In a future PR, I think we can test `crds`, `csv-template`, `mod.check`, and `gen-rbac` on linux in a single action. I think having `codegen` be its own action is good since it takes a lot longer to run. We can maybe do a `git reset --hard HEAD` in between to isolate the changes from each other.

Reply:
Oh right we do have them in both places to verify on mac and linux. A separate action sounds good then to follow that pattern.